jwala-services/src/test/java/com/cerner/jwala/service/jvm/impl/JvmServiceImplTest.java

```diff
@@ -1281,5 +1281,56 @@ public JvmService getJvmService() {
         }
 
     }
+    
+  
+    @Test
+    public void testUpgradeJDKAndDeployJvm() throws CommandFailureException, IOException {
```

Comment:
Move this test up with the rest of the tests instead of at the end after all of the test configuration mock objects.